### PR TITLE
fix(issue 256): upload issue with images fix

### PR DESCRIPTION
**Pull request by dependentmadani**
## Problem
The issue #256 where the 'files' is used the upload request instead of `images` in default configuration.

## Cause
The cause of this issue is when the field in the configuration is not defined by the user, for this reason, the 'files' is used by `ajax.transport` by default.

## Solution
The solution for this issue is to use `image` value in config of `ImageTool` when it is not defined by the user.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -99,8 +99,8 @@ export default class ImageTool implements BlockTool {
       endpoints: config.endpoints,
       additionalRequestData: config.additionalRequestData,
       additionalRequestHeaders: config.additionalRequestHeaders,
-      field: config.field,
-      types: config.types,
+      field: config.field || 'image',
```
Comment (neSpecc) [inline src/index.ts:102]: you already have it in `uploader.ts`. Why it is not working?

image

File: src/uploader.ts [hunk 2]
```diff
@@ -89,12 +89,12 @@ export default class Uploader {
       upload = ajax.transport({
         url: this.config.endpoints.byFile,
         data: this.config.additionalRequestData,
-        accept: this.config.types,
+        accept: this.config.types || 'image/*',
         headers: new Headers(this.config.additionalRequestHeaders as Record<string, string>),
         beforeSend: (files: File[]) => {
           preparePreview(files[0]);
         },
-        fieldName: this.config.field,
```
Comment (neSpecc) [inline src/uploader.ts:97]: so we should remove the same from uploader.ts? since it will never called

Comment (dependentmadani) [reply src/uploader.ts:97]: what should be removed?

Comment (neSpecc) [reply src/uploader.ts:97]: sorry, confused

**Review by neSpecc — APPROVED** [review]
update package json pls

**Pull request merged**